jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/naming/StringUtils.java

```diff
@@ -25,7 +25,7 @@ public static String cleanupStr(String name) {
 	 *
 	 * @param name            the name
 	 * @param allowDottedKeys whether we remove the dots or not.
-	 * @return
+	 * @return the cleaned up string
```

Comment:
Thanks for the correction, but that is a comment which does not tell us much more than the name of the method. We should probably just remove it (and also remove the `@param name    the name`).

I tend to be fairly strongly opinionated with comments : if they do not add anything to what is already in the code, they should be removed...